src/ecs/constants.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed
+ * under the Apache License Version 2.0.
+ *
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2021 Datadog, Inc.
+ */
+
+import { Duration } from "aws-cdk-lib";
+import { DatadogECSBaseProps, Cardinality } from "./interfaces";
+
+/**
+ * Default props for the Datadog ECS Fargate construct
+ */
+export const DatadogEcsBaseDefaultProps: DatadogECSBaseProps = {
+  registry: "public.ecr.aws/datadog/agent",
+  imageVersion: "latest",
+
+  site: "datadoghq.com",
```

Comment:
maybe we can keep it empty so it use the default value from the agent, 
or maybe use `siteList[0]` to get the first value

```suggestion
  site: siteList[0],
```

Reply:
I think defaulting to the agent's value is probably safer and more accurate than doing siteList[0] incase anyone ever adds a value to the start of that list.